### PR TITLE
gtk-4: new package

### PR DESCRIPTION
**Pull request by debasishbsws**
This package is require to build the freerdp-3 package.

Related: https://github.com/wolfi-dev/os/issues/25933

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

### CODE REVIEW

File: gtk-4.yaml [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+      - pango-dev
+      - py3-docutils
+      - py3-gi-docgen
+      - py3-packaging
+      - sassc
+      - shaderc
+      - shared-mime-info
+      - smartypants
+      - spirv-headers
+      # TODO: right now shaderc also inclused spirv-tools, causin a conflict need to fix that,
```
Comment (mamccorm) [inline gtk-4.yaml:72]: Not sure i understand this comment - mentions we use our own spirv-tools, but then we've it commented out below, is this still valid?

File: gst-plugins-base.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 package:
```
Comment (mamccorm) [inline gst-plugins-base.yaml:1]: Any context for the changes to this existing package? was there build issues before or were we skipping some components?

File: gtk-4.yaml (hunk 1 above)
Comment (debasishbsws) [reply gtk-4.yaml:72]: Yes, I have created a new package called shaderc which now also include the spirv-tools within it. I will create a Issue to track this and make sure we can use shaderc with our spirv-tools.

File: gst-plugins-base.yaml (hunk 2 above)
Comment (debasishbsws) [reply gst-plugins-base.yaml:1]: Yes, the gl plugins are needed for the gtk-4. Previously we were not including it but now we are.

File: gtk-4.yaml (hunk 1 above)
Comment (debasishbsws) [reply gtk-4.yaml:72]: https://github.com/wolfi-dev/os/pull/26125/files#diff-95912073ffd0aac4ceb05bf2c6ee2eb41fa14676a714207ad5f9970ccb2ac557R25 It needs to be done.

**Review by mamccorm — APPROVED** [review]

**Pull request merged**